src/options.ts

```diff
@@ -80,14 +81,19 @@ function setUpArguments(): { argv: any } {
         .choices("s", ["flat", "filesystem"])
         .default("s", "flat")
 
+        .boolean("q")
+        .alias("q", "singleQuotes")
+        .describe("q", "Use single quotes for paths instead of the default double quotes")
+        .default("q", false)
+
         .version()
         .alias("v", "version")
         .default("v", false)
 
         .boolean("V")
         .alias("V", "verbose")
         .describe("V", "Display additional logging information")
-        .default("D", false);
+        .default("V", false);
 }
 
 export function getOptions(): Options {
```

Comment:
Seeing as the quote character is going to be constant throughout execution of the program, you can evaluate it once here and add it to the options object rather than resolving it each time using `getQuoteCharacter`.